lib/matplotlib/colors.py

```diff
@@ -308,13 +310,15 @@ def _to_rgba_no_colorcycle(c, alpha=None):
     """
     Convert *c* to an RGBA color, with no support for color-cycle syntax.
 
-    If *alpha* is given, force the alpha value of the returned RGBA tuple
-    to *alpha*. Otherwise, the alpha value from *c* is used, if it has alpha
-    information, or defaults to 1.
+    If a valid *alpha* is given, force the alpha value of the returned
+    RGBA tuple to *alpha*. Otherwise, the alpha value from *c* is used,
+    if it has alpha information, or defaults to 1.
 
     *alpha* is ignored for the color value ``"none"`` (case-insensitive),
     which always maps to ``(0, 0, 0, 0)``.
     """
+    if alpha is not None and not 0 <= alpha <= 1:
+        raise ValueError('Alpha must be between 0 and 1, inclusive.')
```

Comment:
```suggestion
        raise ValueError("'alpha' must be between 0 and 1, inclusive")
```
Minor style/consistency issue. Generally in Python exceptions
- there is on trailing period
- Parameters names and types are enquoted in single strings.